src/OrchardCore.Modules/OrchardCore.Lists/Models/ListPartFilter.cs

```diff
@@ -0,0 +1,43 @@
+namespace OrchardCore.Lists.Models
+{
+    public class ListPartFilter
```

Comment:
A `ContainerService` nem tud a `ListPart`-ról, szóval nem is így kellene elnevezni. Hanem legyen mondjuk `ContainedItemOptions`.

Reply:
Rendben átneveztem a class nevét és a belőle létrehozott példányt is